sklearn/metrics/_classification.py

```diff
@@ -1253,6 +1264,15 @@ def f1_score(
     >>> y_pred = [[0, 0, 0], [1, 1, 1], [1, 1, 0]]
     >>> f1_score(y_true, y_pred, average=None)
     array([0.66666667, 1.        , 0.66666667])
+
+    Notes
```

Comment:
I assume that this is the reason for the error. We should have another Notes section.

Reply:
I removed the other notes section